app/assets/javascripts/components/student/student_comments.es6.jsx

```diff
@@ -78,6 +68,43 @@ class StudentComments extends Component {
     };
   }
 
+
+  // --------------------------------------------------
+  // Lifecycle
+  // --------------------------------------------------
+  componentWillMount() {
+    this.setState(StudentCommentsStore.getState());
+  }
+
+  componentDidMount() {
+    this._listener = StudentCommentsStore.listen((state) => this.setState(state));
+    StudentCommentsActions.fetchStudentComments(this.props.id);
+  }
+
+  componentWillUnmount() {
+    StudentCommentsStore.unlisten(this._listener);
+  }
+
+  // --------------------------------------------------
+  // Helpers
+  // --------------------------------------------------
+  showCreateOverlay() {
+    StudentActions.storeOverlay(true, 'create_comment');
```

Comment:
We don't even need this helper anymore, just define it straight in the `jsx` below!
As well, let's use type `createComment` for javascript camel case convention.